.github/workflows/main.yml

```diff
@@ -85,3 +85,28 @@ jobs:
         uses: pypa/gh-action-pypi-publish@release/v1
         with:
           password: ${{ secrets.PYPI_API_TOKEN }}
+
+
+  trigger_endless-key-app:
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/') || startsWith(inputs.tagname, 'v')
+    needs: [build_apps-bundle, build_kolibri-explore-plugin_whl]
+
+    steps:
+      - name: Get tag name as the release version when at a tag
+        if: startsWith(github.ref, 'refs/tags/')
+        run: |
+          echo "VERSION=${{ github.ref_name }}" >> $GITHUB_ENV
```

Comment:
Both this and the step below add `VERSION` to the environment. Which one is supposed to be used?

Reply:
It is according the "if" condition.

* `startsWith(github.ref, 'refs/tags/')` is for a tag push
* `${{ !startsWith(github.ref, 'refs/tags/') && startsWith(inputs.tagname, 'v') }` is for the workflow_call